CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.3.2](https://github.com/codecov/uploader/compare/v0.2.6...v0.3.2) (2022-09-13)
```

Comment:
@drazisil-codecov this is a comparison with v0.2.6. Why?

Reply:
Actually , I just saw that the one prior was wonky as well, thank you for the catch. 😬